### PR TITLE
ppg to capture a single pulse on GDS scope

### PR DESCRIPTION
**Pull request by guinneak198**
script for very rough straightforward acquisition of a single pulse on the GDS scope

### CODE REVIEW

File: examples/simple_onepulse_capture.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    # throw some stuff out beforehand
+    thiscapture = thiscapture["t":(0, None)]["t":(None, 24e6)]
+    thiscapture *= 2
+    thiscapture["t", 0] *= 0.5
+    thiscapture.ift("t")
+    # }}}
+    data = thiscapture
+data.set_prop("programmed_t_pulse_us", t90_pulse_us)
+data.set_units("t", "s")
+data.set_prop("acq_params", config_dict.asdict())
```
Comment (jmfranck) [inline examples/simple_onepulse_capture.py:124]: Set postproc type

**Comment by jmfranck** [comment]
[this PR](https://github.com/jmfrancklab/FLinst/pull/3) must be resolved first

**Comment by jmfranck** [comment]
please see comments in code `# PR`

**Comment by guinneak198** [comment]
I was under the impression we wanted this script as bare bones as possible and so had taken out the functions you mentioned giving the user control of the timescale etc without messing with function parameters. but added back in

File: examples/simple_onepulse_capture.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    thiscapture = gds.waveform(ch=2)
+    assert (
+        np.diff(thiscapture["t"][np.r_[0:2]]).item() < 0.5 / 24e6
+    ), "what are you trying to do, your dwell time is too long!!"
+    # {{{ just convert to analytic here, and also downsample
+    #     this is a rare case where we care more about not keeping
+    #     ridiculous quantities of garbage on disk, so we are going to
+    #     throw some stuff out beforehand
+    thiscapture.ft("t", shift=True)
+    thiscapture = thiscapture["t":(0, None)]["t":(None, 24e6)]
```
Comment (jmfranck) [inline examples/simple_onepulse_capture.py]: why isn't this just `["t":(0,24e6)]`?  If there is a reason, can you add a comment?

**Review by jmfranck — CHANGES_REQUESTED** [review]
just one thing, and then we should be able to merge.
Thanks for getting this to such a good state!

File: examples/simple_onepulse_capture.py (hunk 2 above)
Comment (guinneak198) [reply examples/simple_onepulse_capture.py]: was basing off of other acquisition scriptss but adjusted to (0,24e6) - I agree this way is more straightforward

**Pull request merged**